cristin-import/src/test/java/no/unit/nva/cristin/mapper/CristinMapperTest.java

```diff
@@ -523,6 +526,98 @@ void shouldCreateNullPublisherWhenAllPublisherDataIsMissing() {
         assertThat( book.getPublisher(), is(instanceOf(NullPublisher.class)));
     }
 
+    @Test
```

Comment:
Here are all the tests :)

Reply:
I sense that you are struggling with cucumber?

Reply:
Not struggling, but it takes a bit more time to implement them :)

Reply:
Siden det ikke er meg som skal leve med dette, så skal jeg ikke kreve av deg at du skriver det om som cucumber tester. Men si ifra hvis du ønsker å ha disse som cucumber tester, men du opplever det som tungt.